config.test.yaml

```diff
@@ -150,34 +150,144 @@ templates:
+                          count: 1
+                          grace_ttl: 86400
+                        valueType: 'json'
+                        version: 1
                 x-request-handler:
-                  - get_from_backend:
+                  - handle_req:
                       request:
-                        uri: http://appservice.wmflabs.org/{domain}/v1/page/mobile-html-sections-remaining/{title}
+                        method: get
+                        uri: /{domain}/sys/mobileapps/handling/mobile-html-sections-remaining/{title}
+                        headers:
+                          cache-control: '{cache-control}'
             /text/{title}:
               get:
+                x-setup-handler:
+                  - init:
+                      method: put
+                      uri: /{domain}/sys/key_rev_value/mobileapps.mobile-text
+                      body:
+                        revisionRetentionPolicy:
+                          type: 'latest'
+                          count: 1
+                          grace_ttl: 86400
+                        valueType: 'json'
+                        version: 1
                 x-request-handler:
-                  - get_from_backend:
+                  - handle_req:
                       request:
-                        uri: http://appservice.wmflabs.org/{domain}/v1/page/mobile-text/{title}
+                        method: get
+                        uri: /{domain}/sys/mobileapps/handling/mobile-text/{title}
+                        headers:
+                          cache-control: '{cache-control}'
+            /handling/{route}/{title}:
+              get:
+                x-request-handler:
+                  - get_rev:
+                      request:
+                        method: get
+                        uri: /{domain}/sys/page_revisions/page/{title}
+                        headers:
+                          cache-control: '{cache-control}'
+                  - check_storage:
+                      request:
+                        method: get
+                        uri: /{domain}/sys/key_rev_value/mobileapps.{$.request.params.route}/{$.request.params.title}/{$.get_rev.body.items[0].rev}/{$.get_rev.body.items[0].tid}
+                      return_if:
+                        status: '2xx'
+                      return:
+                        status: 200
+                        headers: '{$.check_storage.body.headers}'
+                        body: '{$.check_storage.body.body}'
+                      catch:
+                        status: 404
+                  - get_mobileapps:
+                      request:
+                        method: get
+                        uri: http://appservice.wmflabs.org/{domain}/v1/page/{$.request.params.route}/{$.request.params.title}
+                        headers:
+                          x-restbase-etag: '{$.get_rev.body.items[0].rev}/{$.get_rev.body.items[0].tid}'
+                  - store:
```

Comment:
What's the `x-restbase-etag` header? Why is it called like this and not like a plain `etag`?